Mathlib/Order/JordanHolder.lean

```diff
@@ -251,49 +197,12 @@ theorem toList_nodup (s : CompositionSeries X) : s.toList.Nodup :=
   s.toList_sorted.nodup
 #align composition_series.to_list_nodup CompositionSeries.toList_nodup
 
-@[simp]
-theorem mem_toList {s : CompositionSeries X} {x : X} : x ∈ s.toList ↔ x ∈ s := by
-  rw [toList, List.mem_ofFn, mem_def]
-#align composition_series.mem_to_list CompositionSeries.mem_toList
-
-/-- Make a `CompositionSeries X` from the ordered list of its elements. -/
-def ofList (l : List X) (hl : l ≠ []) (hc : List.Chain' IsMaximal l) : CompositionSeries X where
-  length := l.length - 1
-  series i := l.get <| i.cast <| by have := List.length_pos_of_ne_nil hl; omega
-  step' := fun ⟨i, hi⟩ => List.chain'_iff_get.1 hc i hi
-#align composition_series.of_list CompositionSeries.ofList
-
-theorem length_ofList (l : List X) (hl : l ≠ []) (hc : List.Chain' IsMaximal l) :
-    (ofList l hl hc).length = l.length - 1 :=
-  rfl
-#align composition_series.length_of_list CompositionSeries.length_ofList
-
-theorem ofList_toList (s : CompositionSeries X) :
-    ofList s.toList s.toList_ne_nil s.chain'_toList = s := by
-  refine ext_fun ?_ ?_
-  · rw [length_ofList, length_toList, Nat.add_one_sub_one]
-  · rintro ⟨i, hi⟩
-    simp [ofList, toList]
-#align composition_series.of_list_to_list CompositionSeries.ofList_toList
-
-@[simp]
-theorem ofList_toList' (s : CompositionSeries X) :
-    ofList s.toList s.toList_ne_nil s.chain'_toList = s :=
-  ofList_toList s
-#align composition_series.of_list_to_list' CompositionSeries.ofList_toList'
-
-@[simp]
-theorem toList_ofList (l : List X) (hl : l ≠ []) (hc : List.Chain' IsMaximal l) :
-    toList (ofList l hl hc) = l := by
-  refine List.ext_get ?_ ?_
-  · rw [length_toList, length_ofList]
-    have := List.length_pos_of_ne_nil hl
-    omega
-  · intro i hi hi'
-    dsimp [ofList, toList]
-    rw [List.get_ofFn]
-    rfl
-#align composition_series.to_list_of_list CompositionSeries.toList_ofList
+#align composition_series.mem_to_list RelSeries.mem_toList
+#align composition_series.of_list RelSeries.fromListChain'
```

Comment:
Here as well I prefer the `CompositionSeries` name. Can you fix the `RelSeries` name in a future PR?

Reply:
Should I just do it in this PR?

Reply:
It might be better to stick to just one job in this PR, it's already +199 lines.